R/translate-sql-cut.R

```diff
@@ -0,0 +1,61 @@
+sql_cut <- function(x, breaks, labels = NULL, right = TRUE) {
+  breaks <- sort.int(as.double(breaks))
```

Comment:
This has the side-effects of stripping any `NA` from `breaks`? Is that deliberate? If so, should add a test.

Reply:
It is what `base::cut()` does but it looks terrible to me... So I'm not completely sure it is a good idea to also do that?

Reply:
If you don't have any strong feelings, I'd suggest adding `na.last = TRUE` to `sort.int` and then erroring if `breaks` contains any NAs. I implemented something similar for scales very recently: https://github.com/r-lib/scales/blob/main/R/label-number.r#L323-L332